src/main/java/com/lpvs/LicensePreValidationSystem.java

```diff
@@ -26,19 +28,29 @@ public class LicensePreValidationSystem {
 
     private int corePoolSize;
 
-    public LicensePreValidationSystem(@Value("${lpvs.cores:8}") int corePoolSize) {
+    private String lpvsVersion;
+
+    private static Logger LOG = LoggerFactory.getLogger(LicensePreValidationSystem.class);
+
+    public LicensePreValidationSystem(@Value("${lpvs.cores:8}") int corePoolSize,
+                                      @Value("${lpvs.version:}") String lpvsVersion) {
```

Comment:
Isn't it possible to read the version from `pom.xml` file?

Reply:
I love the suggestion from @t25kim and this is also due to my personal preference to track down any libraries that we are relying on from `pom.xml` as an universal way. By the way, which version of slf4j that we are going to rely on?

Reply:
@o-kopysov It is very good to print version information. However I concern about managing version information with several files.

Reply:
Agree with your opinion.